docs/guides/execution-modes.mdx

```diff
@@ -5,73 +5,169 @@ description: An overview of the available execution modes in Qiskit Runtime; ses
+## How long do workloads run?
+
+The amount of time an execution runs depends on the values for maximum execution time (or _time to live_ (TTL)), interactive timeout (or _interactive time to live_ (ITTL)), and, for Qiskit Runtime on IBM Cloud&reg;, the specified Cost limit.
+
+**Maximum execution time (TTL)** - Determines how long (in _quantum time_) a session or batch can run.  This is the smaller of the value set for `max_execution_time` and the QPU-determined job timeout value.  Quantum time is the time spent by the QPU complex to process the workload. This timer starts when the first job in the workload starts being processed and continues running until the value is reached. After this value is reached, the workload is terminated, any jobs that are already running continue running, and any queued jobs that remain in the session or batch are put into a failed state. You can set this value with the `max_time` parameter for [batches](/guides/run-jobs-batch#specify-batch-length) or [sessions](/guides/run-jobs-session#specify-length).
+
+**Interactive timeout (ITTL)** - This value cannot be configured. If no jobs are queued for the active session or batch within that window, the workload is temporarily deactivated. A job submitted to the session or batch reactivates the deactivated workload if the maximum timeout value has not been reached.
+
+For full details about these values, including how to determine the ITTL value, review the [Maximum execution time guide.](/guides/max-execution-time)
+
+<Admonition>
+The time your workload actively runs is probably different from the amount of wall clock time a workload takes.
+</Admonition>
+
+## Basic workflow
+
+The basic workflow for batches and sessions is similar:
 
-[**Batch mode**](run-jobs-batch): A multi-job manager for efficiently running an experiment that is comprised of bundles of independent jobs. Use batch mode to submit multiple primitive jobs simultaneously.   To run in batch mode, specify `mode=batch` when instantiating a primitive or run the job in a batch context manager. See [Run jobs in a batch](run-jobs-batch) for examples.
+1. The first job in a batch or session enters the normal queue. Note: for batches, the entire batch of jobs is scheduled together.
+2. When the first job starts running, the TTL timer starts (and does not stop or pause until the end is reached).
+3. The ITTL timer starts after each job is completed.  If there are no workload jobs ready within the ITTL window, the workload is temporarily deactivated and normal job selection resumes. A job can reactivate the deactivated workload if the batch or session has not reached its TTL value.
+   <Admonition type="note">
+   The job must go through the normal queue to reactivate the workload.
+   </Admonition>
+3. If the TTL value is reached, the workload ends and any remaining queued jobs fail. However, any jobs already running will run to completion.
 
```

Comment:
After reading this section, I feel the previous one (`How long do workloads run?`) can be merged into this one. This section describes the TTLs better (even with a video!). The only thing missing is what TTL stands for, and the note about cost limit on ibm cloud.